wdl/tasks/AlignedMetrics.wdl

```diff
@@ -51,16 +51,16 @@ workflow AlignedMetrics {
 
     call FilterMQ0Reads { input: bam = aligned_bam }
 
-#    call CollectSamErrorMetrics {
-#        input:
-#            bam = aligned_bam,
-#            bai = aligned_bai,
-#            ref_fasta = ref_fasta,
-#            ref_dict = ref_dict,
-#            dbsnp_vcf = dbsnp_vcf,
-#            dbsnp_tbi = dbsnp_tbi,
-#            region = metrics_locus,
-#    }
+    call CollectSamErrorMetrics {
```

Comment:
We should leave this commented out (I was asked to omit it by GP as it was just taking too long to run on every single workflow invocation).  A future PR that I'm working on right now will delete it entirely.

Reply:
OK, sounds good.